src/_listNavigationItem.scss

```diff
@@ -0,0 +1,352 @@
+@import './listDefinitions.scss';
+
+$fd-list-navigation-item-height: 2.75rem !default;
+$fd-list-navigation-item-indicator-spacing: 0.3125rem !default;
+$fd-list-navigation-item-text-spacing: 2.5rem !default;
+
+.#{$block} {
+  list-style-type: none;
+  background: var(--sapList_AlternatingBackground);
+
+  .#{$block}__navigation-item-indicator {
+    @include fd-reset();
+
+    &::after {
+      content: '';
+      width: 0.1875rem;
+      display: inline-block;
+      position: absolute;
+      right: 0;
+      top: 0.5625rem; // (height of nav item - height of indicator) / 2
```

Comment:
Maybe use math function instead of hardcoded value, in case later we change variable values?

Reply:
done